### PR TITLE
add guardian doc and update gov doc

### PR DESCRIPTION
**Pull request by chengwenxi**

### CODE REVIEW

File: docs/features/governance.md [hunk 1]
```diff
@@ -164,3 +164,20 @@ Detail in [Upgrade](upgrade.md)
 * `threshold` default: 0.5, scope（0，1）
 * `governance_penalty` default: 0.667, scope（0，1）
 *  Vote rules:If the ratio of all voters' `voting_power` to the total 'voting_power' in system less than “participation”, the proposal won't be passed. If the ratio of strongly opposed `voting_power` to all voters' `voting_power` more than “veto”, the proposal won't be passed. Then if the ratio of approved `voting_power` to all voter's `voting_power` except abstentions over “threshold”, the proposal will be passed. Otherwise, N/A.
+
+
+### Proposals on community funds usage
+There are three usage, `Burn`, `Distribute` and `Grant`. `Burn` means burn tokens from community funds. `Distribute` and `Grant` will transfer tokens to the destination trustee account from community funds and then trustee will distribute or grant this tokens to others.
```
Comment (shirleypyj) [inline docs/features/governance.md]: three usages

Comment (shirleypyj) [inline docs/features/governance.md]: `Burn` means burning tokens；
transfer tokens to the target trustee's account;
trustee will distribute or grant these tokens to others.

File: docs/features/guardian.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Guardian User Guide
+
+## Introduction
+IRISnet introduce two types of privileged system user controlled by foundations, the profiler and the trustee. 
+
+* Profiler privileges
+    1. Submit software upgrade/halt proposal by governance.
+    2. Initiate service invocation using profiling mode，profiling mode would be exempt from service fees.
```
Comment (shirleypyj) [inline docs/features/guardian.md]: Invocate a service by profiling mode, under which service fees can be exempted.

File: docs/features/guardian.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    
+* Trustee privileges
+    1. As the destination address of a `TxTaxUsage` proposal if usage type is `Distribute` or `grant`.
```
Comment (shirleypyj) [inline docs/features/guardian.md]: To be the target address if the usage type of a `TxTaxUsage` proposal is `Distribute` or `grant`.

File: docs/features/guardian.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    2. Send transaction to withdraw coins to a account from system service fee tax pool.
```
Comment (shirleypyj) [inline docs/features/guardian.md]: an account;

File: docs/features/guardian.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    iriscli guardian profilers
+    ```
+    Query trustee list
+    ```shell
+    iriscli guardian trustees
+    ```
+    
+3. Profiler submit software upgrade/halt proposal
+
+    Detail in [upgrade](upgrade.md)
```
Comment (shirleypyj) [inline docs/features/guardian.md]: Details

File: docs/features/guardian.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+4. Profiler initiate service invocation using profiling mode
```
Comment (shirleypyj) [inline docs/features/guardian.md]: invovate a service

File: docs/features/guardian.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+    ```shell
+    iriscli service call --def-chain-id=[def-chain-id] --service-name=[service-name] --method-id=[method-id] --bind-chain-id=[bind-chain-id] --provider=[provider address] --service-fee=1iris --request-data=[request-data] --chain-id=[chain-id] --from=[key name] --fee=0.004iris
+    ```
+    
+5. Trustee as the destination address of `TxTaxUsage` proposal
```
Comment (shirleypyj) [inline docs/features/guardian.md]: to be the target address

File: docs/features/guardian.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+    Detail in [governance](governance.md#proposals-on-community-funds-usage)
```
Comment (shirleypyj) [inline docs/features/guardian.md]: Details

**Comment by chengwenxi** [comment]
@shirleypyj  Thanks a lot!

**Pull request closed without merging**